static/js/publisher/release/releasesController.js

```diff
@@ -60,6 +63,39 @@ class ReleasesController extends Component {
     this.props.updateReleases(releasesData.releases);
   }
 
+  fetchSetDefaultTrack(track) {
```

Comment:
Ideally we should avoid adding any new functionality to the controller as this was mainly used in pre-redux implementation.

Fetches can be done inside actions. This way they can be triggered by any component without needing to pass reference to this handler via props through components tree.

Reply:
:+1: I did it separate to start with, but wanted to follow the pattern already there. Will move it back out :D

Reply:
Thanks!